convert_video_to_frames.py

```diff
@@ -0,0 +1,12 @@
+import os
+import shutil
+
+def convert_video_to_frames(input_path, output_path, fps=None):
+    if os.path.exists(output_path):
+        print(f'Deleting directory {output_path}')
```

Comment:
Instead of deleting without asking, can we have the program break here with an error message, so that the user can remove or rename the output_path themselves?

Reply:
My reasoning was that these are auto-generated files, so the user could easily regenerate them if they were deleted accidentally (similarly to how a compiler doesn’t ask before replacing the compiled output file). Specifically, the directory generated by `convert_video_to_frames` seems like an internal detail of the program that the user shouldn’t need to manually delete between runs.

But as I think more about it, I can see scenarios where the user might manually modify the output, so you’re probably right that it’s safer not to delete without asking.

Reply:
I agree that these are temporary files. What if:

- There is a `-f` flag (force) that enables your implementation
- If the flag isn't specified, it instead increments the directory name (instead of storing in `./data/`, store in `./data_1/`, or `./data_2/` if that exists, and so on)